packages/wdio-types/src/Capabilities.ts

```diff
@@ -1082,6 +1082,7 @@ export interface BrowserStackCapabilities {
     browserName?: string
     browserVersion?: string
     acceptSslCerts?: boolean
+    wdioService?: string
```

Comment:
This probably should be marked as `@private`
```suggestion
    wdioService?: string
```